### PR TITLE
ISSUE-90722 Change SRS docker version compatible with Pega 8.6

### PR DESCRIPTION
**Pull request by pega-kshev**
Changed the SRS version and Elasticsearch versions to be used with Pega 8.6 to the latest compatible versions.
Updated all corresponding document references.

### CODE REVIEW

File: charts/backingservices/values.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ srs:
 # based on helm charts defined at https://github.com/elastic/helm-charts/tree/master/elasticsearch and may be modified
 # as per runtime and storage requirements.
 elasticsearch:
-  imageTag: 7.1.1
+  imageTag: 7.9.0
```
Comment (vanga) [inline charts/backingservices/values.yaml:44]: Can we use latest patch version available from ES?

**Review by taz-pega-work — APPROVED** [review]

**Review by taz-pega-work — APPROVED** [review]
These changes are ideal. thanks for making the changes in the runbooks!

File: charts/backingservices/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The service deployment provisions runtime service pods along with a dependency o
 Pega Infinity version   | SRS version   | ElasticSearch version     | Description
 ---                     | ---           | ---                       | ---
 < 8.6                   | NA            | NA                        | SRS can be used with Pega Infinity 8.6 and later
-\>= 8.6                 | \>= 1.9.0     | 7.1.x                    | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image that is tagged with version 1.9.0 and later. Current SRS versions are certified to support Elasticsearch version 7.1.x.
+\>= 8.6                 | \>= 1.12.0-2     | 7.9.x                    | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image that is tagged with version 1.12.0 and later. Current SRS versions are certified to support Elasticsearch version 7.9.x.
```
[the remark this answers is not in the document]
Comment (dcasavant) [reply charts/backingservices/README.md:24]: This should be `1.12.0`, correct?  No build number should be included here.

[the remark this answers is not in the document]
Comment (pega-kshev) [reply charts/backingservices/README.md]: yes @dcasavant we were working on retagging it to without build#. Changed this file once the docker image was retagged.

**Review by dcasavant — APPROVED** [review]

**Pull request merged**